apps/pigment-next-app/README.md

```diff
@@ -13,6 +13,8 @@ pnpm build:zero
 Then start the Next.js development server:
 
 ```bash
+cd apps/
+pnpm install
```

Comment:
@siriwatknp I've updated the command, but can you share why this was implemented as a separate workspace? It complicates the running, we need to make sure we commit the changes after running pnpm install in the apps/directory etc.

Reply:
Using the same `workspace` does not give the same result as targeting the `build` folder directly.

We need to target the `build` folder to get closer to the real scenario where `@mui/material` is bundled into a package.

This let me find bugs like https://github.com/orgs/mui/projects/27/views/1?pane=issue&itemId=55489502